keymap.c

```diff
@@ -815,6 +816,29 @@ static const char *km_keyname(int c)
   return buf;
 }
 
+/**
+ * mutt_init_abort_key - Parse the abort_key config string
+ *
+ * Parse the string into C_AbortKeyStr and put the keycode into AbortKey.
+ */
+void mutt_init_abort_key(void)
+{
+  keycode_t buf[2];
+  size_t len = parsekeys(C_AbortKeyStr, buf, sizeof(buf) / sizeof(buf[0]));
```

Comment:
We can simplify this with the `mutt_array_size()` macro.

Reply:
Should be addressed in 21d22cd81d3493a3464f9240471432369999dc04